src/main/kotlin/no/nav/dagpenger/inntekt/Configuration.kt

```diff
@@ -27,7 +31,11 @@ private val devProperties = ConfigurationMap(
                 "database.name" to "dp-inntekt-db-preprod",
                 "vault.mountpath" to "postgresql/preprod-fss/",
                 "application.profile" to "DEV",
-                "application.httpPort" to "8099"
+                "application.httpPort" to "8099",
+                "hentinntektliste.url" to "https://localhost/inntektskomponenten-ws/rs/api/v1/hentinntektliste",
```

Comment:
You might consider removing these "default" config values  from DEV. The application will then fail fast with a nice error message if these config values are missing from the environment.

Reply:
good point..

Reply:
fixed